website/docs/docs/dbt-cloud/setting-up-dbt-cloud/connect-to-your-database/redshift-and-postgres.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Connecting to Redshift and Postgres"
+id: "connecting-to-redshift-and-postgres"
+sidebar_label: "Redshift and Postgres"
+---
+
+## Connecting to Redshift and Postgres
+
+The following fields are required when creating a Redshift connection:
+
+| Field | Description | Examples |
+| ----- | ----------- | -------- |
+| Host Name | The hostname of the Postgres or Redshift database to connect to. This can either be a hostname an IP address. | `xxx.us-east-1.amazonaws.com` |
```

Comment:
```suggestion
| Host Name | The hostname of the Postgres or Redshift database to connect to. This can either be a hostname or an IP address. | `xxx.us-east-1.amazonaws.com` |
```